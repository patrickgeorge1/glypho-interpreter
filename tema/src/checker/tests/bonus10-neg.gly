ttte'UeU"sesM'K'HHy2im*maz{z%FFFg###