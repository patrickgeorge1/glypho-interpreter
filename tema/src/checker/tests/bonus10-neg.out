1
S12DYER57YHSDF89GH7DER12UY3WFB8S9D789W75E123REY856DF74897TERY432HDFS1SDF6H5SD4
